selftests/unit/test_spawner.py

```diff
@@ -2,16 +2,16 @@
 import unittest
 
 from avocado.core import nrunner
+from avocado.core.spawners.process import ProcessSpawner
```

Comment:
Maybe the changes on this file are part of a different commit? I can't see how this fits on the commit description.

Reply:
The `is_task_alive()` method for the `ProcessSpawner` is fixed, as per the commit message.  And this introduces the testing of the `ProcessSpawner`, including the `is_task_alive()` behavior.  Is it clear?

Reply:
What I mean is, the resulting `selftests.unit.test_spawner.Process.test_never_spawned` introduced here, will run `ProcessSpawner.is_task_alive()`.

Reply:
@clebergnu ok, makes sense.